File: android/MvpMoxyRxModule/~!Module_name!~Presenter.kt
package ~!package!~

import com.arellomobile.mvp.InjectViewState
import com.arellomobile.mvp.MvpPresenter
import com.arellomobile.mvp.MvpView


interface ~!Module_name!~View : MvpView {


}


@InjectViewState
class ~!Module_name!~Presenter(private val router: ~!Module_name!~RouterImp) : MvpPresenter<~!Module_name!~View>() {


    override fun onFirstViewAttach() {
        super.onFirstViewAttach()
    }
}

File: ios/View MvpModule/~!module_name!~Presenter.swift
import Foundation
import RxSwift


protocol ~!module_name!~View: class {

}

class ~!module_name!~Presenter {

    private weak var view: ~!module_name!~View!
    private let router: ~!module_name!~Router
    

    init(_ view: ~!module_name!~View, _ router: ~!module_name!~Router) {
        self.view = view
        self.router = router
    }

    func viewDidLoad() {
        
    }
}
